furnace.c

```diff
@@ -230,6 +258,34 @@ command_handler(furnace_context_t* ctx, uint8_t* buffer, void (*feedback)(const
     }
   }
 #endif
+#if CONFIG_WATER
+  else if(sscanf(buffer, "water %u", &arg) == 1) {
+    const int pwm_value = arg + WATER_OFFSET;
+    if(pwm_value < 0){
+      const char msg[] = "water pwm argument too small!\r\n";
+      const size_t msg_len = sizeof(msg)-1;
+      feedback(msg, msg_len);
+    }
```

Comment:
The documentation says we are expecting value in range `<0:10>`, so let's check for that.